packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -378,6 +411,20 @@ export function TransferPanelMain({
     setQueryParams({ l2ChainId })
   }, [isConnectedToArbitrum, externalFrom, externalTo, setQueryParams])
 
+  useEffect(() => {
+    // If a different destination address is specified, we want to update the ERC-20 balance accordingly
+    if (selectedToken && isAddress(destinationAddressOrWalletAddress)) {
+      updateErc20L1Balance([selectedToken.address])
+      updateErc20L2Balance([String(selectedToken.l2Address)])
+    }
+  }, [
+    destinationAddressOrWalletAddress,
+    isDepositMode,
```

Comment:
do we need `isDepositMode` in this dep array?

Reply:
I thought we did but after testing again we don't